doc/faq.md

```diff
@@ -57,6 +57,65 @@ directory. None of this should affect any existing Haskell tools at all.
   there may be more options here for new projects in the future (see issue
   [253](https://github.com/commercialhaskell/stack/issues/253))
 
+### Cabal-the-library
+
+The relationship between stack and Cabal-the-library is complicated.
+
+1. Stack itself builds against a version of the Cabal library, which it uses for parsing Cabal files.
+
+2. Stack wraps functionality from a version of the Cabal library in order to build packages.
```

Comment:
"wraps" may be slightly confusing. Stack builds a `Setup.hs` file against a version of the Cabal library and then shells out to that executable. Stack itself does not perform any of the configure/build/copy/register stuff in-process. This allows it to more easily support newer versions of the Cabal library for performing builds, which is why _typically_ (but not always) you can use a Stack executable with a newer version of GHC than it was built for.

Reply:
I've used your language to fix.